README.md

```diff
@@ -48,26 +41,31 @@ The hardware for the KFD-AVR fork is very simple. You can either purchase a KFDS
 
 ![schematic](doc/pic/basic_hw_schematic.png)
 
-Where PIN3 is Arduino pin 3, PIN5 is Arduino pin 5, DATA is the radio data line, SNS, is the radio sense line, and GND is the radio ground line. VCC is 5 volts (make sure you are using a 5 volt tolerant board).
+Where PIN3 is Arduino pin 3 for receive data, PIN5 is Arduino pin 5 for transmit data, DATA is the radio data line, SNS, is the radio sense line, and GND is the radio ground line. VCC is 5 volts (make sure you are using a 5 volt tolerant board).
 
-If you are using a shield, on the TRS connector, tip is data, ring is sense, and shield is ground.
+On the 3.5mm TRS connector, tip is DATA, ring is SNS, and shield is GND.
+
+⚠️ KFDmini (eBay clones) cables and accessories are NOT compatible with KFD-AVR 3.5mm TRS jacks/pinouts, and vice-versa. 

```

Comment:
Ehhhhh please rephrase this, since KFD-AVR is this repo, there's no standard TRS jacks for KFD-AVR, it's just that y'all happen to have used 14mm and they used 17mm, neither of which is related to the protocol/software stack.